### ISSUE TITLE
Mining Docs Feedback 

### ISSUE THREAD

**Issue by GinaAbrams** [blockstack/docs#929]
Starting a thread here to share feedback received from Xan on the [Technical Specifications](https://docs.blockstack.org/stacks-blockchain/technical-specs) page: 

- Docs highlight SIP-007. I think this is mostly up to date, but we should think about removing the numbers mentioning how many liquid stacks there are and roughly what the token threshold to Stack is, as that will change constantly, and setting an expectation there around an exact number could be problematic.
- Right now the further explanation of how PoX mining works is somewhat split between SIP-001 and SIP-007. I find this confusing. We should consider fully writing out how PoX mining works in SIP-007, even if it includes reproducing some of the same copy that is in SIP-001. At a minimum, the docs should also link to SIP-001 for further details on how PoX mining works instead of just SIP-007.
- No mention of microblocks under block interval.  
- To confirm on reward maturity window: Miners aren't sent the STX until 100 blocks later? Just want to confirm whether it's a delay on the transaction, or if the tokens are transferred immediately but a lock for 100 blocks.
- General comment about PoX mining: Mining is actually pretty complex when you get into the details. Smoothing function, microblocks, mining support transactions, fair mining pools etc...  Looking for a summary of all the different parts of STX Mining, as we do for Stacking.

**Comment by diwakergupta** [comment]
> Starting a thread here to share feedback received from Xan on the Technical Specifications page:

Thanks for the feedback @xanbots!  I'm not seeing any concrete action items from this issue, responses inline.

> * Docs highlight SIP-007. I think this is mostly up to date, but we should think about removing the numbers mentioning how many liquid stacks there are and roughly what the token threshold to Stack is, as that will change constantly, and setting an expectation there around an exact number could be problematic.
> * Right now the further explanation of how PoX mining works is somewhat split between SIP-001 and SIP-007. I find this confusing. We should consider fully writing out how PoX mining works in SIP-007, even if it includes reproducing some of the same copy that is in SIP-001. At a minimum, the docs should also link to SIP-001 for further details on how PoX mining works instead of just SIP-007.

I'd decouple changes to the SIPs: they serve a different purpose and target a different audience. For instance, does https://docs.blockstack.org/stacks-blockchain/proof-of-transfer address the gaps identified above. If not, how can we improve that?

> No mention of microblocks under block interval.

The microblocks mining PR is in review (https://github.com/blockstack/stacks-blockchain/pull/2085) -- I can include details on microblocks once that's merged.

> To confirm on reward maturity window: Miners aren't sent the STX until 100 blocks later? Just want to confirm whether it's a delay on the transaction, or if the tokens are transferred immediately but a lock for 100 blocks.

Former: miners earn the coinbase only after the maturity window has elapsed.

> General comment about PoX mining: Mining is actually pretty complex when you get into the details. Smoothing function, microblocks, mining support transactions, fair mining pools etc... Looking for a summary of all the different parts of STX Mining, as we do for Stacking.

Can you clarify the ask here? Is it to expand https://docs.blockstack.org/stacks-blockchain/mining to cover more details, or include more mining info in the technical specs page?

**Comment by agraebe** [comment]
wanted to bump up this conversation - any next steps here? who can help with a PR if needed?

**Comment by xanbots** [comment]
I really like the "understand mining" page, and think the only comment is to continue to expand on the information that is there as new PR's are merged. Micro blocks and the smoothing function are two topics I'd like to see more about. Thanks!

**Comment by agraebe** [comment]
Got it, thanks! Closing this issue and will track upcoming work in respective new issues
